tapdb/migrations.go

```diff
@@ -146,17 +182,43 @@ func applyMigrations(fs fs.FS, driver database.Driver, path, dbName string,
 			ErrMigrationDowngrade, migrationVersion, latestVersion)
 	}
 
-	log.Infof("Applying migrations from version=%v", migrationVersion)
+	// Compute the max migration version known to the driver.
+	maxMigrationVersion, err := maxKnownMigrationVersion(migrateFileServer)
```

Comment:
Can't we just use the `LatestMigrationVersion` constant that is checked to be up to date with a CI script?

Reply:
Good idea. Latest commit use `latestVersion` from
```
latestVersion := opts.latestVersion.UnwrapOr(LatestMigrationVersion)
```
which is basically the same as what you've suggested, I think `opts.latestVersion` is just there for tests. But I thought I'd point that out just in case I was missing something.